climakitae/core/data_export.py

```diff
@@ -199,17 +229,50 @@ def _export_to_netcdf(data, save_name):
         data to export to NetCDF format
     save_name : string
         desired output file name, including the file extension
+    mode : string
+        localtion logic for storing export file.
 
     Returns
     -------
     None
 
     """
-    est_file_size = _estimate_file_size(data, "NetCDF")
-    disk_space = shutil.disk_usage("./")[2] / bytes_per_gigabyte
+    print("Exporting specified data to NetCDF...")
+
+    # Convert xr.DataArray to xr.Dataset so that compression can be utilized
+    _data = data
```

Comment:
I just realized that this might create a whole other copy of the data and would cause memory to double?

Reply:
Hmm, I will look further into this in another PR